contracts/vc/VVVVCTokenDistributor.sol

```diff
@@ -44,6 +45,7 @@ contract VVVVCTokenDistributor is VVVAuthorizationRegistryChecker {
         @param signature Signature of the user's KYC wallet address
      */
     struct ClaimParams {
+        address callerAddress;
```

Comment:
Correct, not great. I renamed `callerAddress` to `tokenRecipient` and transfer the tokens to `callerAddress` instead of `msg.sender` just to be more explicit about it.

This way the input argument is used, and there's just a check to ensure it matches `msg.sender`